operator/v1/types_network.go

```diff
@@ -137,6 +141,17 @@ type OpenShiftSDNConfig struct {
 	UseExternalOpenvswitch *bool `json:"useExternalOpenvswitch,omitempty"`
 }
 
+// KuryrConfig configures the Kuryr-Kubernetes SDN
+type KuryrConfig struct {
+	// The port kuryr-daemon will listen for readiness and liveness requests.
+	// +optional
+	DaemonProbesPort *uint16 `json:"daemonProbesPort,omitempty"`
```

Comment:
Let's try to keep it consistent with other ports, such as:
- https://github.com/openshift/api/blob/master/operator/v1/types_network.go#L127
- https://github.com/openshift/api/blob/master/openshiftcontrolplane/v1/types.go#L223
- https://github.com/openshift/api/blob/master/operator/v1/types_network.go#L127
similarly in k8s API ports are usually 32-bit ints. I'm fine with this being uint,  but please make it 32-bit. Same applies to the other one.

Reply:
Done, thanks!